kubernetes/apps/overlays/gtfs-rt-archiver-v3-prod/archiver-channel-vars.yaml

```diff
@@ -6,6 +6,5 @@ data:
   AIRFLOW_ENV: cal-itp-data-infra
   CALITP_BUCKET__GTFS_DOWNLOAD_CONFIG: "gs://calitp-gtfs-download-config"
   CALITP_BUCKET__GTFS_RT_RAW: "gs://calitp-gtfs-rt-raw-v2"
-  HUEY_CONSUMER_WORKERS: "16"
```

Comment:
Thanks for the offline walkthrough, only remaining question is why we're removing this / why set to 4 (instead of 16) in the docker-compose below... Not sure I am following that part